client/src/pages/event/SingleEvent.js

```diff
@@ -0,0 +1,182 @@
+              <br />
+              <span>
+                {address.city}&nbsp;{address.state}&nbsp;{address.zip}
+              </span>
+            </p>
+          </section>
+          <section className='w-full p-4 bg-white flex items-center text-sm text-center'>
+            <button
+              className='inline-block flex-1 bg-transparent hover:bg-green-500 text-green-500 hover:text-white border-r border-gray-200 font-semibold py-2'
+              style={{ outline: 'none' }}
+            >
+              <CheckBoxOutlinedIcon fontSize='small' />
+              &nbsp; Accept
+            </button>
+            <button
+              className='inline-block flex-1 bg-transparent hover:bg-red-500 text-red-500 hover:text-white font-semibold py-2'
+              style={{ outline: 'none' }}
+            >
+              <CancelPresentationIcon fontSize='small' />
+              &nbsp; Reject
+            </button>
+          </section>
+          <section className='px-6 py-4 bg-white border-b border-gray-400'>
+            <h4 className='text-sm text-gray-600 font-light uppercase mb-2'>
+              Workers, Chat, & Photos
+            </h4>
+            <Link
+              to={participantListPath(id)}
+              className='flex justify-between items-center w-full bg-white p-2 text-sm font-light'
+            >
+              <div>
+                <small className='block text-gray-600'>Participants:</small>
+                <p>0 Invited, 0 Attending</p>
+              </div>
+              <KeyboardArrowRightIcon />
+            </Link>
+            <Link
+              to='!#'
+              className='flex justify-between items-center w-full bg-white p-2 text-sm font-light'
+            >
+              <div>
+                <small className='block text-gray-600'>Event Chat: (off)</small>
+                <p>0 Messages</p>
+              </div>
+              <KeyboardArrowRightIcon />
+            </Link>
+            <Link
+              to='!#'
+              className='flex justify-between items-center w-full bg-white p-2 text-sm font-light'
+            >
+              <div>
+                <small className='block text-gray-600'>Photos:</small>
+                <p>0 Photos</p>
+              </div>
+              <KeyboardArrowRightIcon />
+            </Link>
+          </section>
+          <section className='w-full p-4 border-t border-b border-gray-400 my-1 bg-white'>
+            <h4 className='text-sm text-gray-600 font-light uppercase mb-2'>
+              Event Details
+            </h4>
+            <Link
+              to='!#'
+              className='bg-transparent hover:bg-transparent text-teal-500 text-xs'
+            >
+              Add Event Details
+            </Link>
+          </section>
+          <section className='w-full p-4 border-t border-b border-gray-400 my-1 bg-white'>
+            <h4 className='text-sm text-gray-600 font-light uppercase mb-2'>
+              Links
+            </h4>
```

Comment:
Should display list of links associated with this event

Modify `src/constants/events.json` to include links array

Reply:
fixed in 7dfc58c2b04eaf322f76756a82d9b96c488e7ff5